doc/Core.xml

```diff
@@ -1270,9 +1286,13 @@
       </section>
       <section>
         <title>Firmware Upgrade</title>
-        <para>Two mechanisms are provided for upgrading the firmware on a device. The first uses the UpgradeSystemFirmware command to send the new firmware image using MTOM.</para>
```

Comment:
Shouldn't we deprecate MTOM based firmware upgrade?

Reply:
I have no objection for this. I propose we just share it during the next F2F meeting.